Activities/Database/UiPath.Database.Activities/DatabaseExecute.cs

```diff
@@ -49,6 +49,7 @@ public abstract class DatabaseExecute : AsyncTaskCodeActivity
         [LocalizedCategory(nameof(Resources.Input))]
         [LocalizedDisplayName(nameof(Resources.Activity_DatabaseExecute_Property_Parameters_Name))]
         [LocalizedDescription(nameof(Resources.Activity_DatabaseExecute_Property_Parameters_Description))]
+        [Browsable(true)]
```

Comment:
I don't understand this change? Is there a behavior in place in Studio that hides Dictionary properties or something else?
I mean this should do nothing, the property should be visible by default 😕

Reply:
I'm not sure
This attribute was deleted by mistake for this property some time ago so I just reverted this change

Reply:
It seems to override another hidden property... In a cross-platform project, if I rename the property (and its associated DesignProperty), it will be created in another position, suggesting that somewhere there's a hidden Parameters property declaration with the Others/Misc category.

Original:
![Original](https://github.com/user-attachments/assets/efe1ffb9-068f-4bc8-86f6-771333350298)

Renamed:
![Renamed](https://github.com/user-attachments/assets/2330319d-ea80-489e-821f-2ad4ea75bc5c)